x-pack/plugins/triggers_actions_ui/public/application/sections/rule_form/rule_edit.tsx

```diff
@@ -213,7 +213,7 @@ export const RuleEdit = ({
         aria-labelledby="flyoutRuleEditTitle"
         size="m"
         maxWidth={620}
-        ownFocus={false}
```

Comment:
I'm wondering why `ownFocus={false}` [was added](https://github.com/elastic/kibana/commit/d91c6d0cfba6c006975f89f77ad90510ce2ef81c#diff-d4a953ef9323a22cb24a9250faaaa3da69a0c7334ac39458f731d275ceda4429) in the first place 🤔 
Being able to interact with the rest of the page outside of the flyout doesn't seem too useful in this case...

Reply:
Yea I had the same thought. Did a git blame and turns out it was added 3 years ago.